### PR TITLE
Relative version of FocalRescale & MaskToCrop

### LINKED ISSUE #1: Hard limits for the focal point
At the moment there is a hard limit on the focal point of 2048 which is far too low if normal images out of a camera are being used. I would suggest to strip such restriction.
Even better, I feel like the focal point should not be expressed in terms of pixels, but as a ratio over the original image so that it is scale invariant.

### PR DESCRIPTION
**Pull request by KaruroChori**
I added support for a more advanced node to crop and rescale. It works with relative coordinates, and allows for smaller than 100% crops of the original image. It also returns important information about the sampled window in respect to the original image. This is useful for workflows where we later need to merge it back somehow.

I am also working on a second node `MaskToCrop` to use in conjunction with the alternative FocalRescale.  
Basically it finds the centroid and bounding box of the mask, and it passes it to adapt the cropping.

When completed & if accepted, this also closes https://github.com/Felsir/Comfy_Felsirnodes/issues/1.

### CODE REVIEW

**Review by Felsir — APPROVED** [review]

**Comment by Felsir** [comment]
Thank you for improving and adding to these nodes!
The relative node has crossed my mind in the past, as images got more diverse. I am going to experiment further with this.

**Comment by KaruroChori** [comment]
Hi thank you for merging it! However, I tagged it as `draft` because I was still finishing it :D.
All main functionalities were done and somewhat tested by the last commit you merged, but the documentation for the nodes I added was not completed yet.

Also, I tested for the first time the publishing process for the comfy registry & added the relevant configuration to the repository. At the moment it is configured to use my namespace & secret token, so some changes are needed on your end if you want to publish official versions under your name.

I will be opening a new PR once I am able to finish the documentation of the new nodes to make it at the same level of the rest. And to fix any bug I might encounter while further using the new nodes.

**Comment by Felsir** [comment]
No worries, I'll check the next PR. 

I wasn't familiar with the publishing process so I only noticed after the merge that it threw an error. I'll have a look later to see what needs changing. I'll check the documentation of the registry, so far I haven't looked into it because I had the impression noone would actually use my nodes except me :-)

**Comment by Felsir** [comment]
Took a bit of tinkering, but I got the publish in a working state.
